pyproject.toml

```diff
@@ -30,10 +31,9 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
-SQLAlchemy = ">=1.4.17,<=1.4.41"
+python = "^3.7"
+SQLAlchemy = ">=2.0.0,<=2.0.5.post1"
```

Comment:
Can we remove ```<=2.0.5.post1```? I tested it with the latest version and it works fine.

Reply:
When I started this PR it was the latest version, I haven't tested it with more recent versions but will do soon.

Reply:
use `SQLAlchemy = ">=2.0.0,<=2.1"`  ?